potassium-nitrite/src/test/kotlin/org/dizitart/kno2/KotlinXSerializationMapperTest.kt

```diff
@@ -43,7 +45,7 @@ class KotlinXSerializationMapperTest {
         val someMap: Map<String, String>,
         val someSerializableObjectMap: Map<InnerObject, SomePolymorphType>,
         val innerObject: InnerObject,
-        val id: String,
+        @SerialName("_id") val id: String? = null,
```

Comment:
I would recommend not to name any user defined field as "_id". It is reserved to store the `NitriteId` in a document. 

If you want to get hold of the `NitriteId`, just use the datatype as `NitriteId` and mark the field with `@Id` annotation, any field name would be ok. 

If not, keep the field name anything other than "_id" in the document.

Reply:
Got it!